drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c

```diff
@@ -221,6 +234,63 @@ static int lsm6dsv16x_accel_range_set(const struct device *dev, int32_t range)
 	return 0;
 }
 
+static int lsm6dsv16x_accel_wake_threshold_set(const struct device *dev,
+					       const struct sensor_value *val)
+{
+	const struct lsm6dsv16x_config *cfg = dev->config;
+	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
+	lsm6dsv16x_act_thresholds_t thresholds;
+
+	if (lsm6dsv16x_act_thresholds_get(ctx, &thresholds) < 0) {
+		LOG_DBG("failed to get thresholds");
+		return -EIO;
+	}
+
+	float val_mg = sensor_ms2_to_ug(val) / 1000.0f;
+
+	if (val_mg < (7.8125f * 63.0f)) {
```

Comment:
Can you `#define` these constants so they're not inline?

Reply:
Reworked it